doc/systest_basics.rst

```diff
@@ -0,0 +1,288 @@
+vector between the two points, that is, :math:`g = \gamma_t + i\gamma_x`.
+
+Point-shear estimates are equivalent to average tangential shear, returned as real
+:math:`\langle \gamma_t \rangle` and imaginary :math:`\langle \gamma_x \rangle`.  If random
+catalogs are given, they are used as random lenses, and
+:math:`\langle \gamma_t \rangle-\langle \gamma_{t, {\rm rand}} \rangle` is returned instead.
+
+Shear-shear correlation functions are :math:`\xi_+` and :math:`\xi_-`.  :math:`\xi_+` is,
+nominally, *g1* times *g2\**, and is given as both the real and imaginary components.
+:math:`\xi_{+,im}` should be consistent with 0 to within noise. :math:`\xi_-` on the other hand is
+*g1* times *g2* (not complex conjugate).  Similarly, :math:`\xi_{-,im}` should be 0.  (Note that
+*g1* and *g2* here are two *complex* shears :math:`g_{1t} + ig_{1x}` and :math:`g_{2t} + ig_{2x}`
+from the two catalogs, not the two components of a single shear in sky coordinates or chip frame.)
+
+Point-scalar (point-kappa) estimates are equivalent to <scalar>; scalar-shear estimates are
+equivalent to <scalar*Re(shear)> with a corresponding imaginary case; scalar-scalar estimates
+are <scalar1*scalar2>.  Random catalogs result in compensated estimators as in the point-shear
+case.
+
+Whisker plots
+-------------
+
+Whisker plots are child classes of :class:`stile.sys_tests.BaseWhiskerPlotSysTest`.  They're the
+standard weak lensing whisker plots: visualizations of the shear field using headless arrows.
+The predefined types are:
+
+- **Star**: whisker plot of shapes of PSF stars
+- **PSF**: whisker plot of PSF shapes at the location of PSF stars
+- **Residual**: whisker plot of (star shape-PSF shape)
+- **None**: an empty :class:`stile.sys_tests.BaseWhiskerPlotSysTest` class instance, which can be
+  used for multiple types of whisker plots.  See the documentation for
+  :class:`stile.sys_tests.BaseWhiskerPlotSysTest` (especially the method
+  :func:`whiskerPlot <stile.sys_tests.BaseWhiskerPlotSysTest.whiskerPlot>`) for more details.
+  Note that this type has a different call signature than
+  the other methods and that it lacks many of the convenience variables the other
+  WhiskerPlots have, such as self.objects_list and self.required_quantities.
+
+Calls to child classes of :class:`stile.sys_tests.BaseWhiskerPlotSysTest`
+return :class:`matplotlib.figure.Figure` instances and can be saved directly as images, but
+a :func:`.plot` method is included (that just returns the figure passed to it) so the plotting
+interface is the same as other :class:`SysTests <stile.sys_tests.SysTest>`.
+
+Scatter plots
+-------------
+
+Scatter plots are child classes of :class:`stile.sys_tests.BaseScatterPlotSysTest`.  They generate
+scatter plots of the data plus a linear regression fit through the scattered points.  There is
+also an optional comparison trendline at :math:`y=0` or :math:`x=y` (or a user-defined function).
+The predefined types are:
+
+- **StarVsPSFG1**: star vs PSF g1
```

Comment:
Yeah, g1/g2--this is just comparing estimates of the same objects so there's no obvious direction to choose as t and x, unlike for the spatial correlations.  Should we highlight this?